frontend/src/pages/RecurringRunDetails.tsx

```diff
@@ -170,8 +171,8 @@ class RecurringRunDetails extends Page<{}, RecurringRunConfigState> {
     const pageTitle = run ? run.name! : runId;
 
     const toolbarActions = [...this.props.toolbarProps.actions];
-    toolbarActions[1].disabled = !!run.enabled;
-    toolbarActions[2].disabled = !run.enabled;
+    toolbarActions[2].disabled = !!run.enabled;
```

Comment:
We can probably improve upon the practice of accessing toolbar items by index.

Reply:
Yes, I will work on a follow-up PR for this

Reply:
Follow-up: #1812